biogeophys/FatesHydroWTFMod.F90

```diff
@@ -321,7 +323,6 @@ function th_from_psi_base(this,psi) result(th)
     class(wrf_type)     :: this
     real(r8),intent(in) :: psi
     real(r8)            :: th
-    th = 0._r8
```

Comment:
These should not be removed, this will cause compile problems for XLF if the base does have  a result.